review-demo.rb

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env ruby
+
+# Add the lib directory to Ruby's load path so our requires will work.
+$:.unshift(File.expand_path('./lib'))
+
+require 'happo'
+
+system 'open', Happo::Utils.construct_url('/review-demo')
+require 'happo/server'
```

Comment:
Did you consider making this part of the cli tool instead? `happo review-demo` or something.